src/component/Home.jsx

```diff
@@ -0,0 +1,373 @@
+                            boxShadow: 2,
+
+                            display: "flex",
+                            justifyContent: "center",
+                          }}
+                          variant="outlined"
+                        >
+                          {values.address}
+                        </Card>
+                      </td>
+                      <td>
+                        <Card
+                          sx={{
+                            width: 150,
+                            padding: 2,
+                            boxShadow: 2,
+                            display: "flex",
+                            justifyContent: "center",
+                          }}
+                          variant="outlined"
+                        >
+                          {values.email}
+                        </Card>
+                      </td>
+                      <td>
+                        <Card
+                          sx={{
+                            width: 150,
+                            padding: 2,
+                            boxShadow: 2,
+
+                            display: "flex",
+                            justifyContent: "center",
+                          }}
+                          variant="outlined"
+                        >
+                          {values.phoneno}
+                        </Card>
+                      </td>
+                      <td>
+                        <Card
+                          sx={{
+                            width: 80,
+                            padding: 2,
+                            boxShadow: 2,
+
+                            display: "flex",
+                            justifyContent: "center",
+                          }}
+                          variant="outlined"
+                        >
+                          <DeleteIcon
+                            sx={{ color: "red" }}
+                            onClick={() => handleDelete(values.id)}
+                          />
+                        </Card>
+                      </td>
+
+                      <td>
+                        <Card
+                          sx={{
+                            width: 80,
+                            padding: 2,
+                            boxShadow: 2,
+
+                            display: "flex",
+                            justifyContent: "center",
+                          }}
+                          variant="outlined"
+                        >
+                          <EditIcon
+                            sx={{
+                              color: "blue",
+                            }}
+                            onClick={() => handleEdit(values.id)}
+                          />
+                        </Card>
+                      </td>
+                    </tr>
+                  ))}
+                  </tbody>
+                </table>
+              </Box>
+            </Stack>
+          </Box>
+          <Stack
```

Comment:
what is the use of this stack ?

Reply:
stack is the mui container arranging elements vertically or horizontally.